src/index.js

```diff
@@ -0,0 +1,178 @@
+let temperatureLabel = null;
+let temperatureIncreaseButton = null;
+let temperatureDecreaseButton = null;
+let getRealtimeTemperatureButton = null;
+let skyImage = null;
+let landscapeImage = null;
+let cityNameInput = null;
+let cityNameResetButton = null;
+let skySelectValue = null;
+let gardenBackground = null;
+let temperatureValue = null;
+let defaultCityName = null;
+
+const showTemp = () => {
+    temperatureLabel.textContent = temperatureValue;
+}; 
+
+const showCity = () => {
+    cityNameLabel.textContent = defaultCityName;
+    cityNameInput.value = defaultCityName;
+}; 
+
+const setTempLabelColor = (temperature) => {
+    if (temperature >= 80) {
+        return temperatureLabel.setAttribute("class", "red");
+    } else if (temperature >= 70) {
+        return temperatureLabel.setAttribute("class", "orange");
+    } else if (temperature >= 60) {
+        return temperatureLabel.setAttribute("class", "yellow");
+    } else if (temperature >= 50) {
+        return temperatureLabel.setAttribute("class", "green");
+    } else {
+        return temperatureLabel.setAttribute("class", "teal");
+    }
+};
+
+const landscapeImages = {
+    80: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    70: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+    60: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+    50: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+};
+
+const skyImages = {
+    Sunny: "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+    Cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+    Rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+    Snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+};
+
+const setLandscapeImage = temperature => {
+    if (temperature >= 80) {
+        return landscapeImages[80];
+    } else if (temperature >= 70) {
+        return landscapeImages[70];
+    } else if (temperature >= 60) {
+        return landscapeImages[60];
+    } else {
+        return landscapeImages[50];
+    }
+};
+
+const setSkyImage = skyDescription => {
+    return skyImages[skyDescription];
+};
+
+const setGardenBackgroundColor = (skyDescription) => {
+    if (skyDescription === "Sunny") {
```

Comment:
If we packed the sky class into a record along with the sky emoji, we could look up the appropriate class directly rather than using a brittle chain of `if`/`else if` conditions.